src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -141,9 +142,13 @@ private void runNoteInIntakeState() {
       intakeState = IntakeState.NOTE_IN_INTAKE_AND_KICKER;
       this.transitionGamePiece();
     } else if (!inputs.isRollerIRBlocked) {
-      intakeState = IntakeState.NOTE_IN_BETWEEN_INTAKE_AND_KICKER;
+      if (quickShootingEnabled) {
```

Comment:
This conditional won't execute if the previous condition is true. This may be why the feature wasn't working as expected.

I recommend checking the quickShootingEnable variable first.